sdk-setup/src/mb2

```diff
@@ -78,9 +76,9 @@ the "build" or "qmake" command:
 
 Limitations related to shadow builds:
 
-    1. Shadow build is only available for packages that use qmake and
-       their build and install procedure is fully realized using qmake
-       with no additional steps implemented at .spec level.
+    1. Shadow build is only available for packages that use qmake (or CMake)
+       and their build and install procedure is fully realized using qmake (or
+       CMake) with no additional steps implemented at .spec level.
```

Comment:
I would drop the parenthesis and break the long sentence:
`1. Shadow build is only available for packages that use qmake or CMake. Their build and install procedure must be fully realized using qmake or CMake, with no additional steps implemented at .spec level.`

Reply:
In future there will be also autotools and possibly others, so I think it is better to formulate the common parts for the "preferred" option and only indicate the alternate options this way.

Reply:
We will have to modify the text anyway at that point

Reply:
I mean at that point we would just extend it but not restructure, e.g.,

 "...that uses qmake (or CMake or GNU Autotools) and their..."

and where possible (as in the latter case here) just some shortened form like

 "...realized using qmake (CMake etc.) with..."

Reply:
I get what you mean, but I think we should anyway try to improve the readability of the text. Having long sentences and more and more text inside parenthesis does not exactly make it easy to read and comprehend. It's not (yet) a disaster, but anyway it has room for improvement.

Reply:
I am worried about that too. I was able to split the sentence after all.

Reply:
Looks good, thanks :)